### PR TITLE
fix/M2-3300-scores-report-tooltip

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ import { ForbiddenScoreIdSymbols, scoreIdBase } from './ScoreContent.const';
 export const getTableScoreItems = (items: Item[]) =>
   items?.map((item) => ({
     id: getEntityKey(item),
-    name: item.name,
+    name: `${item.question}`,
```
Comment (wildkiomi) [inline src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts:19]: redundant `${}`

File: src/shared/components/ContentWithTooltip/ContentWithTooltip.tsx [hunk 2]
```diff
@@ -6,13 +6,18 @@ import { Tooltip } from 'shared/components/Tooltip';
...
+  styles = {},
+  tooltipByDefault,
+}: ContentWithTooltipProps) => {
   const { width } = useWindowSize();
   const elementRef = useRef<HTMLDivElement>(null);
   const hasTooltip = useIsTextNodeEllipsed(elementRef, [elementRef.current, width, value]);
 
   return (
-    <Tooltip placement={'top'} tooltipTitle={hasTooltip ? value : undefined}>
+    <Tooltip placement={'top'} tooltipTitle={tooltipByDefault || hasTooltip ? value : undefined}>
```
Comment (wildkiomi) [inline src/shared/components/ContentWithTooltip/ContentWithTooltip.tsx:20]: remove pls brackets for _placement="top"_

File: src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts (hunk 1 above)
Comment (kdanilenka32) [reply src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts:19]: need for type

File: src/shared/components/ContentWithTooltip/ContentWithTooltip.tsx (hunk 2 above)
Comment (kdanilenka32) [reply src/shared/components/ContentWithTooltip/ContentWithTooltip.tsx]: fixed

**Pull request merged**